### PR TITLE
Add i18n markers to new and some existing components (fix #56)

### LINKED ISSUE #56: Strings are missing i18n markers
In some of the newly-added components, a lot of strings are missing i18n markers (`t()`, `nt()` and so on), so they are effectively not ready for localization.

Among others, this includes `HelpDialogContent` and `AboutDialogContent`.

### PR DESCRIPTION
**Pull request by iafan**
@julen, please review

### CODE REVIEW

File: pootle/static/js/browser/components/ProgressBar.js [hunk 1]
```diff
@@ -7,6 +7,7 @@
  */
 
 import React from 'react';
+import { t } from 'utils/i18n';
```
Comment (julen) [inline pootle/static/js/browser/components/ProgressBar.js]: Nit: internal imports are grouped separately from library imports.

File: pootle/static/js/shared/components/AboutDialog.js [hunk 2]
```diff
@@ -31,22 +32,25 @@ const AboutDialogContent = React.createClass({
           <h1>Zing</h1>
 
           <p>
-            This server is powered by <strong>Zing</strong> —
+            {tct(`This server is powered by %(zing)s —
```
Comment (julen) [inline pootle/static/js/shared/components/AboutDialog.js]: Note this is not a template string, and regular quotes should be used instead. Also note that the word wrapping would expose extra whitespace after extraction, so it should go in a single line.

The same applies to _KeyboardHelpDialog_.

File: pootle/static/js/shared/components/AboutDialog.js [hunk 3]
```diff
@@ -31,22 +32,25 @@ const AboutDialogContent = React.createClass({
...
+              evernoteLink: <a href="https://evernote.com/">Evernote</a>,
+              pootleLink: <a href="http://pootle.translatehouse.org/">Pootle</a>,
+            })}
           </p>
 
           <p>
-            Source code and bug tracker:
-            {' '}<a href="https://github.com/evernote/zing">Github</a>
+            {tct('Source code and bug tracker: %(githubLink)s', {
+              githubLink: <a href="https://github.com/evernote/zing">Github</a>,
```
Comment (julen) [inline pootle/static/js/shared/components/AboutDialog.js]: Typo: Github -> GitHub

**Comment by julen** [comment]
GTM provided the comments are addressed.

File: pootle/static/js/browser/components/ProgressBar.js (hunk 1 above)
Comment (iafan) [inline pootle/static/js/browser/components/ProgressBar.js]: Fixed

File: pootle/static/js/browser/components/ProgressBar.js [hunk 4]
```diff
@@ -29,28 +30,39 @@ const ProgressBar = React.createClass({
...
+        t('%(percent)s untranslated', {
+          percent: `<span class="value untranslated">${pUntransNumber}%</span>`,
+        }),
+        '<br/>'
+      );
     }
 
     return (
-      <table className="graph" title={title}>
+      <table className="graph" title={titleParts.join('')}>
```
Comment (iafan) [inline pootle/static/js/browser/components/ProgressBar.js]: Fixed

File: pootle/static/js/shared/components/AboutDialog.js (hunk 2 above)
Comment (julen) [inline pootle/static/js/shared/components/AboutDialog.js]: Given that we use that in some other places, I'm guessing it should work, although I haven't tested that myself. JS strings are extracted into a master resource file by leveraging `xgettext` (run `make pot` for running the script that does everything extraction-related).

**Pull request merged**